convert_tendl_rand.py

```diff
@@ -0,0 +1,356 @@
+#!/usr/bin/env python3
+
+import argparse
+import glob
+import os
+import sys
+import tarfile
+from pathlib import Path
+from multiprocessing import Pool
+
+import openmc.data
+
+from utils import download
+
+description = """
+Download random TENDL libraries from PSI and convert it to a HDF5 library for use with OpenMC. 
+Only certain nuclides are available from PSI. This script generates a cross_sections_tendl.xml 
+file with random TENDL evaluations plus a standard library located in 'OPENMC_CROSS_SECTIONS'
+"""
+
+
+class CustomFormatter(
+    argparse.ArgumentDefaultsHelpFormatter, argparse.RawDescriptionHelpFormatter
+):
+    pass
+
+
+n_choices = [
+    "all",
+    "O16",
+    "Si28",
+    "Si29",
+    "Si30",
+    "Fe54",
+    "Fe56",
+    "Fe57",
+    "Fe58",
+    "Na23",
+    "Pu240",
+]
+
+parser = argparse.ArgumentParser(
+    description=description, formatter_class=CustomFormatter
+)
+parser.add_argument("-n", "--nuclides", choices=n_choices, nargs="+",
+                    default="Fe56", help="The nuclides to be downloaded. Available are: "
```

Comment:
```suggestion
                    default=["Fe56"], help="The nuclides to be downloaded. Available are: "
```
Without this, the script currently fails if you let it use the default